js/src/components/List.js

```diff
@@ -78,7 +78,7 @@ const ListElement = ({ items, type }) => (
               <StyledIcon fontSize="small">brightness_1</StyledIcon>
             </ListItemIcon>
 
-            <ListItemText>
+            <ListItemText data-anu-lms-list-item-text>
```

Comment:
can we use `data-test="anu-lms-list-item-text"` instead? See https://docs.cypress.io/guides/references/best-practices#Real-World-Example for example